prisma/schema.prisma

```diff
@@ -0,0 +1,101 @@
+datasource db {
+  provider = "mysql"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model User {
+  userId           Int              @id @default(autoincrement())
+  name             String
+  email            String
+  type             UserType
+  skills           Skill[]
+  jobApplications  JobApplication[]
+  jobPostings      JobPosting[]
+  contacts         User[]           @relation("contacts")
+  password         String
+  city             String?
+  state            String?
+  resume           Resume?
+  contactsRelation User[]           @relation("contacts")
```

Comment:
I don't know enough about prisma relations to know if this is correct. Is this field necessary since it's identical to line 18?

Reply:
Leaving this PR for you to merge btw